### PR TITLE
User, School & Timetable Admin: increase feild lengths in forms and database

### PR DESCRIPTION
**Pull request by SKuipers**
This PR increases the field length for a number of fields in the system. These changes have been running in production for over a year with no issues, finally got around to PRing 😄 

The name lengths were particularly important for out school: some nationalities have particularly long names, or multiple first & last names. For houses, one school is using colours and the other animal names, 4 chars wasn't enough (even for a meaningful abbreviation).

`gibbonPerson`
surname → 60
firstName → 60
preferredName → 60
nameInCharacters → 60
profession → 90
employer → 90
job title →90
emergency1name → 90
emergency2name → 90

`gibbonCourse`
name → 60
nameShort → 12

`gibbonHouse`
nameShort → 10

### CODE REVIEW

File: modules/Timetable Admin/course_manage_add.php [hunk 1]
```diff
@@ -82,15 +82,15 @@
...
+				$row->addTextField('name')->isRequired()->maxLength(60);
 			
 			$row = $form->addRow();
 				$row->addLabel('nameShort', __('Short Name'));
 				$row->addTextField('nameShort')->isRequired()->maxLength(8);
 			
 			$row = $form->addRow();
 				$row->addLabel('orderBy', __('Order'))->description(__('May be used to adjust arrangement of courses in reports.'));
-				$row->addNumber('orderBy')->maxLength(6);
+				$row->addNumber('orderBy')->maxLength(12);
```
Comment (rossdotparker) [inline modules/Timetable Admin/course_manage_add.php]: I think you got the wrong field here. This is orderBy, not nameShort

File: modules/Timetable Admin/course_manage_edit.php [hunk 2]
```diff
@@ -99,15 +99,15 @@
...
+				$row->addTextField('name')->isRequired()->maxLength(60);
 
 			$row = $form->addRow();
 				$row->addLabel('nameShort', __('Short Name'));
 				$row->addTextField('nameShort')->isRequired()->maxLength(8);
 
 			$row = $form->addRow();
 				$row->addLabel('orderBy', __('Order'))->description(__('May be used to adjust arrangement of courses in reports.'));
-				$row->addNumber('orderBy')->maxLength(6);
+				$row->addNumber('orderBy')->maxLength(12);
```
Comment (rossdotparker) [inline modules/Timetable Admin/course_manage_edit.php]: Same here...

**Comment by rossdotparker** [comment]
I've left two comments in the code, as I think you have the wrong fields. What about updating the matching values in application forms and data updater?

**Comment by SKuipers** [comment]
Thanks, well caught. Pushed the fix for orderBy, and applied the field length changes to data updater & application form.

**Pull request merged**